hack/pin-deps.sh

```diff
@@ -29,7 +29,11 @@ fi
 
 edit_args=(
     -fmt
+    # TODO remove this when sigs.k8s.io/apiserver-builder-alpha is updated to use Kubernetes 1.16.0 
+    # https://github.com/kubernetes-sigs/apiserver-builder-alpha/issues/433
+    -replace sigs.k8s.io/apiserver-builder-alpha=github.com/cofyc/apiserver-builder-alpha@kubernetes-1.16.0-alpha.0
```

Comment:
the official sigs.k8s.io/apiserver-builder-alpha repo does not update to use Kubernetes 1.16 code yet, so I forked it update the code to use Kubernetes 1.16 code

Reply:
Maybe you can send a PR to apiserver-builder-alpha repo so we can update this once the PR is merged.

Reply:
already sent a PR (https://github.com/kubernetes-sigs/apiserver-builder-alpha/pull/434), it's merged just now, I'm going to remove this.

Reply:
done